library/trans.c

```diff
@@ -838,6 +838,8 @@ static void *dnet_reconnect_process(void *data)
 		clock_gettime(CLOCK_MONOTONIC_RAW, &ts2);
 
 		timeout = n->check_timeout - (ts2.tv_sec - ts1.tv_sec);
+		if (!timeout)
+			timeout = 1;
```

Comment:
It should be explained

Reply:
It's my old idea. I think I must remove this, because reconnect_thread doesn't anyway block io_pool, so It's not problem it here would't be any delay.